src/app/shared/components/login-form/login-form.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { LoginFormComponent } from '@shared/components';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterLink, RouterModule, Routes } from '@angular/router';
+import { SharedModule } from '@shared/shared.module';
+
+const loginRoutes: Routes = [
+  {
+    path: '',
+    component: LoginFormComponent,
+  },
+];
+
+@NgModule({
+  declarations: [LoginFormComponent],
+  imports: [
+    RouterModule.forChild(loginRoutes),
```

Comment:
please create the routing for the module

Reply:
I did it, including the ones from above.